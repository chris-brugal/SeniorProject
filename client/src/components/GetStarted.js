import React from "react";
import { Typography, Grid, Button } from "@mui/material";
import "../styles/LandingPage.css";
import { Link as ScrollLink } from "react-scroll";

const GetStarted = () => {
  return (
    <div id="GetStarted" className="home-div">
      <Typography
        className="main-text"
        sx={{
          fontWeight: 525,
          paddingLeft: {
            xs: "0px",
            md: "30px",
          },
          paddingTop: {
            xs: "30px",
            md: "0px",
          },
        }}
        variant="h2"
      >
        Get Started
      </Typography>
      <Grid
        container
        spacing={1}
        alignItems="center"
        justifyContent="center"
        style={{ minHeight: "90vh" }}
      >
        <Grid item xs={12} md={7} container justify="center" alignItems="left">
          <img
            src="/images/city.jpg"
            className="main-img"
            height="50%"
            width="50%"
            alt="join"
          />
        </Grid>
        <Grid
          item
          xs={12}
          md={5}
          container
          justify="center"
          alignItems="center"
        >
          <Typography className="main-text" variant="h2">
            Log in and go! <br />
            <ScrollLink
              activeClass="active"
              to="navbar"
              spy={true}
              smooth={true}
              duration={500}
            >
              <Button variant="contained">Back to the top</Button>
            </ScrollLink>
          </Typography>
        </Grid>
      </Grid>
    </div>
  );
};

export default GetStarted;
