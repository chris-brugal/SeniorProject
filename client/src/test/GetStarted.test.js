import React from 'react';
import GetStarted from '../components/GetStarted';
import renderer from "react-test-renderer";

describe("snapshot testing", () => {

test("snapshot testing for Get Started component", () => {
    const comp = renderer.create(<GetStarted />).toJSON()
    expect(comp).toMatchSnapshot()
})
})