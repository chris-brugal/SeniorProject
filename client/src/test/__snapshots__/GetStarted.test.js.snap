// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing snapshot testing for Get Started component 1`] = `
<div
  className="home-div"
  id="GetStarted"
>
  <h2
    className="MuiTypography-root MuiTypography-h2 main-text css-lrs2p1-MuiTypography-root"
  >
    Get Started
  </h2>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 css-z4xi2a-MuiGrid-root"
    style={
      Object {
        "minHeight": "90vh",
      }
    }
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-7 css-fim4zn-MuiGrid-root"
      justify="center"
    >
      <img
        alt="join"
        className="main-img"
        height="50%"
        src="/images/city.jpg"
        width="50%"
      />
    </div>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-5 css-sc7rel-MuiGrid-root"
      justify="center"
    >
      <h2
        className="MuiTypography-root MuiTypography-h2 main-text css-1sra7t5-MuiTypography-root"
      >
        Log in and go! 
        <br />
        <a
          onClick={[Function]}
        >
          <button
            className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-sghohy-MuiButtonBase-root-MuiButton-root"
            disabled={false}
            onBlur={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            Back to the top
            <span
              className="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </a>
      </h2>
    </div>
  </div>
</div>
`;
